package.json

```diff
@@ -24,6 +24,7 @@
     "@babel/core": "^7.13.14",
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-typescript": "^7.13.0",
+    "@chainlink/external-adapter": "^0.2.6",
```

Comment:
Was an error that you encountered that prompted this to be added?
Or why is this getting added as a dependency here?

Reply:
I don't remember manually adding that so I'm honestly not sure, should I just delete it?